flax/linen/recurrent.py

```diff
@@ -500,6 +500,107 @@ def num_feature_axes(self) -> int:
     return 1
 
 
+class MGUCell(RNNCellBase, metaclass=RNNCellCompatibilityMeta):
+  r"""MGU cell.
+
+  The mathematical definition of the cell is as follows
+
+  .. math::
+
+      \begin{array}{ll}
+      f = \sigma(W_{if} x + b_{if} + W_{hf} h) \\
+      n = \tanh(W_{in} x + b_{in} + f * (W_{hn} h + b_{hn})) \\
+      h' = (1 - f) * n + f * h \\
+      \end{array}
+
+  where x is the input and h, is the output of the previous time step.
+
+  Attributes:
+    features: number of output features.
+    gate_fn: activation function used for gates (default: sigmoid).
+    activation_fn: activation function used for output and memory update
+      (default: tanh).
+    kernel_init: initializer function for the kernels that transform
+      the input (default: lecun_normal).
+    recurrent_kernel_init: initializer function for the kernels that transform
+      the hidden state (default: initializers.orthogonal()).
+    bias_init: initializer for the bias parameters (default: initializers.zeros_init())
+    dtype: the dtype of the computation (default: None).
+    param_dtype: the dtype passed to parameter initializers (default: float32).
+  """
+  features: int
+  gate_fn: Callable[..., Any] = sigmoid
+  activation_fn: Callable[..., Any] = tanh
+  kernel_init: initializers.Initializer = default_kernel_init
+  recurrent_kernel_init: initializers.Initializer = initializers.orthogonal()
```

Comment:
interesting, can you point to the source for this?

Reply:
I copied it over from the GRU class' [`recurrent_kernel_init`](https://github.com/google/flax/pull/3418/files#diff-32512a9b0d5285a594cec41f03de1a4530fd5920847ca3c095fb9d0f0677ae23R423).